.github/workflows/go.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.18
+          go-version: 1.19
```

Comment:
Can we use 1.20?

Reply:
@1995parham jan, it's look like ```quic-go``` has some problems on ```v1.20```.

```shell
quic-go@v0.31.1/internal/qtls/go120.go:5:13: cannot use "The version of quic-go you're using can't be built on Go 1.20 yet.
```